velox/common/memory/MemoryUsageTracker.h

```diff
@@ -203,7 +203,7 @@ class MemoryUsageTracker
     {
       std::lock_guard<std::mutex> l(mutex_);
       auto newUsed = usedReservation_ += size;
-      auto newCap = std::max(minReservation_, newUsed);
+      auto newCap = std::max(minReservation_.load(), newUsed);
```

Comment:
nit: don't need load() call here.

Reply:
Discussed offline, this is needed to allow std::max to infer the template type